### PR TITLE
chore: Apollos CLI package

### PR DESCRIPTION
**Pull request by vinnyjth**
from the apollos cli directory 

`npm exec apollos-cli secrets -d (whatever)` 

or 

- from apollos-cli `npm link` 
- from templates/apollos-church-api `npm exec apollos-cli secrets -d (whatever)`

### CODE REVIEW

**Comment by vinnyjth** [comment]
don't merge yet, found a bug ;)

**Comment by redreceipt** [comment]
didn't work for me

Screen Shot 2021-06-16 at 4 11 17 PM
Screen Shot 2021-06-16 at 4 11 24 PM

**Comment by vinnyjth** [comment]
Uh weird. It’s the ‘npm exec’ that’s failing. I’ll check my npm version in a sec

File: packages/apollos-cli/package.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@apollosproject/apollos-cli",
+  "version": "2.20.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\""
+  },
+  "bin": {
+    "apollos-cli": "./index.js"
```
Comment (redreceipt) [inline packages/apollos-cli/package.json:10]:
```suggestion
    "apollos": "./index.js"
```

Comment (redreceipt) [reply packages/apollos-cli/package.json:10]: cleaner, i think the other libs do it this way

**Comment by redreceipt** [comment]
ah, `npx` worked. did `npm exec` replace it in a newer version?

**Comment by redreceipt** [comment]
so freaking cool

Screen Shot 2021-06-16 at 4 25 18 PM

**Review by redreceipt — APPROVED** [review]

**Comment by vinnyjth** [comment]
Oh shoot @redreceipt - I think it has to be named `apollos-cli`. Because of how `npx` works the package name has to match the `bin` name. Hold off auto-merging until I can confirm.

**Comment by redreceipt** [comment]
It doesn’t I checked

**Comment by vinnyjth** [comment]
you the man - where is that from?

**Comment by redreceipt** [comment]
Screen Shot 2021-06-16 at 4 31 38 PM

**Comment by redreceipt** [comment]
npm docs

**Comment by redreceipt** [comment]
I was looking up how react native does it

**Pull request merged**